xbmc/application/ApplicationStackHelper.h

```diff
@@ -47,7 +47,7 @@ class CApplicationStackHelper : public IApplicationComponent
   /*!
   \brief Returns true if Application is currently playing an ISO stack
   */
-  bool IsPlayingISOStack() const;
+  bool IsPlayingDiscStack() const;
```

Comment:
As you changed the name of the method, you should also adjust the documentation.

Reply:
Done.

Reply:
Done